pkgs/development/php-packages/systemd/default.nix

```diff
@@ -17,6 +18,13 @@ buildPecl {
     hash = "sha256-UhpE9QXChqKLBqutpQ2Y6neo/ULlJGNojf9DCYJfQMU=";
   };
 
+  patches = [
+    (fetchurl {
+      url = "https://github.com/systemd/php-systemd/pull/8.diff";
+      hash = "sha256-g73qI6gKpj5keaGIXdujcii8IWSSBUK9h8gUUVNcKVc=";
+    })
```

Comment:
This should be `fetchpatch` and a comment explaining why this patch is required must be added.

Reply:
Okay, the patche consists of two commits, that are QoL improvements and do not have to be added, but would be nice:
- [1/2] https://github.com/systemd/php-systemd/pull/8/commits/0f25ec9aab7747e85445a48213020ea6adda3658 provides arginfo, which would surpress the *warning* about missing arginfo.
- [2/2] https://github.com/systemd/php-systemd/pull/8/commits/23575461b8cc55fa9c4132a58393b6438c2aff5c would print the location in the php code, not the c code, where `sd_journal_send` was called

Should I remove the patch (and leave it to an overlay) or just add the former comments?